cmplibrary/src/main/java/com/sourcepoint/cmplibrary/exception/ExceptionDescriptions.kt

```diff
@@ -0,0 +1,8 @@
+package com.sourcepoint.cmplibrary.exception
+
+class ExceptionDescriptions {
+
+    companion object {
```

Comment:
We already have an object for this purpose ExceptionCodes.kt

Reply:
I see your point, in this case just add the "No Internet connection." into the exception since this is the only place we use it